src/components/Albums/Albums.scss

```diff
@@ -4,12 +4,21 @@
   display: flex;
   flex-direction: column;
 
+  &__progressbar {
```

Comment:
we agreed to remove progressbar as for now

Reply:
Done